### PR TITLE
refac: generate nonce instead of calling Enclave API

### PR DESCRIPTION
**Pull request by lawliet0521LarryK**

### CODE REVIEW

File: packages/core/src/eth/wallet.ts [hunk 1]
```diff
@@ -339,8 +340,7 @@ export abstract class EthLikeWallet extends Wallet<EthTransaction> {
   }
 
   async getNonce(): Promise<BN> {
-    const nonce = await this.client.get<NonceDTO>(`${this.baseUrl}/nonce`);
-    return BNConverter.hexStringToBN(String(nonce.nonce));
+    return BNConverter.hexStringToBN("0x" + randomBytes(32).toString("hex"));
   }
```
Comment (JhChoy) [inline packages/core/src/eth/wallet.ts:344]: I believe `async` is redundant now.

Comment (lawliet0521LarryK) [reply packages/core/src/eth/wallet.ts]: 반영하고, 놓친 부분도 있어서 추가했습니다!

File: packages/core/src/eth/wallet.ts [hunk 2]
```diff
@@ -626,9 +625,8 @@ export class EthUserWallet extends EthLikeWallet {
     this.userWalletData = userWalletData;
   }
 
-  async getNonce(): Promise<BN> {
-    const nonce = await this.client.get<NonceDTO>(`${this.baseUrl}/nonce`);
-    return BNConverter.hexStringToBN(String(nonce.nonce));
+  getNonce(): BN {
+    return BNConverter.hexStringToBN("0x" + randomBytes(32).toString("hex"));
   }
```
Comment (JhChoy) [inline packages/core/src/eth/wallet.ts:630]: Same function with super class, this definition is redundant.

Comment (lawliet0521LarryK) [reply packages/core/src/eth/wallet.ts]: Ok!

**Review by JhChoy — APPROVED** [review]

**Pull request merged**